scipy/stats/_continuous_distns.py

```diff
@@ -9033,6 +9034,101 @@ def _munp(self, n, b):
 
 rice = rice_gen(a=0.0, name="rice")
 
+class irwinhall_gen(rv_continuous):
+    r"""An Irwin-Hall (Uniform Sum) continuous random variable.
+
+    An `Irwin-Hall <https://en.wikipedia.org/wiki/Irwin-Hall_distribution/>`_ 
+    [1]_ [2]_ continuous random variable is the sum 
+    of :math`n` independent standard uniform r.v.'s. 
+    
+    %(before_notes)s
+
+    Notes
+    -----
+    Applications include `Rao's Spacing Test <https://jammalam.faculty.pstat.ucsb.edu/html/favorite/test.htm>`_, 
+    a more powerful alternative to the Rayleigh test 
+    when the data are not unimodal, and radar [3]_.
+
+    Conveniently, the pdf and cdf are the :math:`n`-fold convolution of 
+    the ones for the standard uniform distribution,
+    which are well known to be just cardinal B-splines of degree :math:`n-1` 
+    with knots evenly spaced from :math:`1` to :math:`n`.
+
+    The Bates distribution is simply the Irwin-Hall distribution scaled 
+    by :math:`1/n`: ``bates = irwinhall(n, name="bates", scale=1/n)``
```

Comment:
Does this actually work? I don't know if you can freeze a distribution with a new name.
```python3
from scipy.stats import irwinhall
n = 10
bates = irwinhall(n, name="bates", scale=1/n)
# TypeError: _parse_args() got an unexpected keyword argument 'name'
```

Reply:
Hm. I didn't see any issues. How else do you make parametrically specialized versions of general distributions?

Reply:
> I didn't see any issues.

OK, but please try executing the code to see if you get the same error.

> How else do you make parametrically specialized versions of general distributions?

I'm not immediately seeing a way to do this such that the result object `bates` behaves exactly like a regular distribution object. It is easy to implement a `bates` such that the object is a "frozen" distribution, which is basically what you have.
```python3
bates = irwinhall(n, scale=1/n)
```
There is probably some way to do it, but it is probably not worth figuring out right now. It is worth including this in the notes of the documentation, though.